src/Fantomas/SourceTransformer.fs

```diff
@@ -56,6 +56,9 @@ let rec multiline synExpr =
 
     | Tuple es ->
         List.exists multiline es
+        
+    | App(DotGet(App(DotGet(dg),_), _),_) -> 
+        true
```

Comment:
This leads to change of behavior of things like `let foo = X.F().G().H()`. Better would be to use `autoNlnCheck` in `genExpr`.

Reply:
Not sure where to add autoNlnCheck here.
`should break on . operator and keep indentation` fails again by removing this.